.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,9 @@ Closes #
 Thank you for contributing to ODK Collect!
 
 Before sending this PR, please read
-https://github.com/getodk/collect/blob/master/CONTRIBUTING.md
+https://github.com/getodk/collect/blob/master/docs/CONTRIBUTING.md
 -->
 
-#### What has been done to verify that this works as intended?
```

Comment:
I've switched this for a checklist item so that it's clear that tests are the default and that manual checking would be an exception that should be justified.